### DISCUSSION TITLE
How to search by case name

### DISCUSSION THREAD

**Discussion by legaltextai** [freelawproject/courtlistener#4169] [General]
Is it possible to find a case using case_name in dockets API? case_name is one of the Options in dockets API 
For example, the case_name is "Theodore Johnson" , I would think something like https://www.courtlistener.com/api/rest/v3/dockets/?case_name="Theodore Johnson" ? but it is not working 
Thanks

**Comment by mlissner** [comment]
No, for that you'll want to use the search API. Case names are just not consistent enough that this is something you'd want to do with your database most of the time.

**Comment by legaltextai** [comment]
but with search api, i still can't filter by case_name, can i? it will search based on words similarity in all fields?

**Comment by mlissner** [comment]
You can certainly filter to a specific field? 

![image](https://github.com/freelawproject/courtlistener/assets/236970/47d86130-d57a-4600-b5b9-38d3ed40d413)

**Reply by legaltextai** [reply]
oh my, i forgot i can turn web search into api / curl calls. thank you!
